### PR TITLE
[Currency] Remove currency toggling

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | yes |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: src/Sylius/Bundle/CurrencyBundle/Form/Type/CurrencyCodeChoiceType.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ public function configureOptions(OptionsResolver $resolver)
         $choices = [];
 
         /** @var CurrencyInterface[] $currencies */
-        $currencies = $this->currencyRepository->findBy(['enabled' => true]);
+        $currencies = $this->currencyRepository->findBy([]);
```
Comment (pamil) [inline src/Sylius/Bundle/CurrencyBundle/Form/Type/CurrencyCodeChoiceType.php]: findAll()

File: src/Sylius/Component/Core/spec/Provider/ChannelBasedCurrencyProviderSpec.php [hunk 2]
```diff
@@ -44,19 +44,14 @@ function it_is_a_currency_provider()
     function it_returns_only_channels_enabled_currencies_as_available_ones(
         ChannelContextInterface $channelContext,
         ChannelInterface $channel,
-        CurrencyInterface $enabledCurrency,
-        CurrencyInterface $disabledCurrency
+        CurrencyInterface $enabledCurrency
```
Comment (michalmarcinkowski) [inline src/Sylius/Component/Core/spec/Provider/ChannelBasedCurrencyProviderSpec.php]: `$currency`

**Comment by michalmarcinkowski** [comment]
@lchrusciel please have a look at the failing build.

**Comment by lchrusciel** [comment]
Should be fine

**Comment by michalmarcinkowski** [comment]
👍

**Pull request merged**